### PR TITLE
Enhance the mail to a direct mailing option which use to be a static …

### PR DESCRIPTION
**Pull request by nitinjaiswal7537**
…mail

Hey @garg3133 check this out
Fixes: This PR will fix the static mail address to a direct Mailing  address

# Prosposed Changes
- changed the static mail address

# Screensh
 Updated
:-------------------------:|:-------------------------:
 ** updated screenshot **
Inkedscreencapture-file-C-Users-Hema-Jaiswal-Desktop-JagratiWebApp-templates-home-index-html-2021-03-11-19_41_19_LI

### CODE REVIEW

**Comment by garg3133** [comment]
@nitinjaiswal7537 You can also take the icon inside the anchor tag. Also, the email seems to be a little off towards the bottom whereas it should be ideally aligned at the centre of icon can you fix that too.

One more thing is that on hovering over the link, it becomes white. Orange colour upon hover would look better here.

**Comment by nitinjaiswal7537** [comment]
@garg3133 Taking the icon under anchor tag and aligning the mail address simultaneously is being to achieve.
So, could i separate the two and align them.
Rest all the task have been done.

**Comment by garg3133** [comment]
@nitinjaiswal7537 Try something like this:
```
<a style="display: inline-block;"><i class="fas fa-envelope fa-2x"></i> <span>jagrati@iiitdmj.ac.in</span></a>
```

Also, can you try to change the icons a bit? Like instead of cell-phone icon, a call icon can be used. Same for envelope icon, if you can find any better.

**Comment by nitinjaiswal7537** [comment]
@garg3133 Issues have been enhanced have a look over it

**Comment by AJAYSHARMA2012** [comment]
Screenshot (263)
@nitinjaiswal7537 Getting this error while running your code. May be it's due to some syntax error or something. Kindly have a look at it.

File: templates/home/index.html [hunk 1]
```diff
@@ -12,7 +12,15 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/css/glide.theme.min.css">
 {% endblock %}
+{% style box for email%}
```
Comment (garg3133) [inline templates/home/index.html:15]: This tag doesn't work in Django. You need to make the required changes in `static/home/css/style.css` file. Also, keep in mind to put the style in the correct section (sections are arranged in the order they appear on the home page.

**Comment by nitinjaiswal7537** [comment]
Hey @AJAYSHARMA2012  syntax error has been resolved.

**Comment by nitinjaiswal7537** [comment]
@garg3133  Actually i have to add all that static/home/css/style.css code in new style sheet or something else, i am not able to figure out what needed to be done

**Comment by garg3133** [comment]
> @garg3133 Actually i have to add all that static/home/css/style.css code in new style sheet or something else, i am not able to figure out what needed to be done

If you need to add any CSS for the changes you made, you should add that in the `static/home/css/style.css` file, at the correct position (the file is divided in many sections through comments).

**Comment by nitinjaiswal7537** [comment]
Hey @garg3133 i have been adding the css code to the required location but the changes does not responds to the local browser
so what should i do??

**Comment by garg3133** [comment]
@nitinjaiswal7537 Try using `Ctrl + F5` to refresh the page. The older CSS sometimes gets cached into the browser and thus the changes don't show up.

**Comment by nitinjaiswal7537** [comment]
Hey @garg3133 i have make a copy of static/home/css/styles.css in the templates folder as templates/home/syles.css as in previous case changes were not reflecting on my browser so i can continue with that??

**Comment by garg3133** [comment]
> Hey @garg3133 i have make a copy of static/home/css/styles.css in the templates folder as templates/home/syles.css as in previous case changes were not reflecting on my browser so i can continue with that??

No, templates should only contain HTML files. All other files are placed in the static folder. For the CSS not showing up, keep clearing the cache of your browser, either with `Ctrl + F5` or `Shift + F5`.

**Comment by nitinjaiswal7537** [comment]
@garg3133 Css have been embedded to it correct position

**Comment by garg3133** [comment]
@nitinjaiswal7537 I really don't seem to understand what you're doing. Open the `Files changed` tab and look for yourself, most of the changes I see there don't relate to this PR and don't make any sense.

**Comment by nitinjaiswal7537** [comment]
@garg3133 Actually there was a merge conflict in this PR so in order to rectify that i had allowed the changes so all this changes are  coming from that except that changes in the mailing option .

**Comment by garg3133** [comment]
@nitinjaiswal7537 And I'm guessing that you must have clicked on "Accept Both Changes", right? That's why there are two instances of some changes.

Can you rectify this? Because this PR can't be merged in this state. You can also consider closing this PR and opening a new one from a new branch.

**Pull request closed without merging**